app/query_objects/active_query.rb

```diff
@@ -0,0 +1,11 @@
+class ActiveQuery
+  delegate :all, :each, :for_each, to: :active
+
+  def initialize(model)
+    @relation = model.where(active: true)
+  end
+
+  def active
+    @relation
+  end
+end
```

Comment:
Is this for caching performance or for code readability? It seems like you're mostly doing ActiveQuery.new() which wouldn't have a cached `@relation`. But tbh I'm confused why else we would want a class just to call one class method. What's the advantage of `ActiveQuery.new(Template)` over `Template.active`?